### PR TITLE
Material hubs/sophia

### PR DESCRIPTION
**Pull request by peicasey**
TODO for Casey: get link for Sophia of a previous TAO video 

font size should be 'text-md' (my bad for figma thing)

for transitions do 'duration-200'

overall looks really good :)

### CODE REVIEW

**Comment by peicasey** [comment]
Get rid of mouseLeave to stop it resetting to the Youtube picture + use cover image so that the image always stays the same size.

For the youtube video, we can change to be an uploaded video. Since I haven't uploaded any video to the youtube channel yet, just know it will probably be this one:
https://drive.google.com/file/d/1MyaoIyPLl6Qh8-ywN92JsqOsPjgIzNg6/view?usp=drive_link

**Review by Andrew800 — COMMENTED** [review]
This looks really good!! :DD
- For the YouTube/Google Drive/Github links, I think it would also be good if clicking on the link also opened it in a new tab
- Also, in page.tsx, the apostrophes in lines 23, 29, 35 for "TAO's" aren't normal apostrophes (for me at least) - I don't think this can cause major issues but I wanted to note that since vscode is bullying me about that

File: club-website/src/app/page.tsx
Comment (peicasey) [inline club-website/src/app/page.tsx:34]: image

switch out apostrophes to `&apos;` to make eslint happy

File: club-website/public/youtube.png
Comment (peicasey) [inline club-website/public/youtube.png:1]: We're using [`react-icons`](https://react-icons.github.io/react-icons/) for icons, so would be nice if these images could be removed.

**Review by peicasey — COMMENTED** [review]
overall good, small changes requested -- but also i realize i made the PR LOL so i can't approve it. so after you do that u can just approve yourself 💀

**Review by sophiaaphu — APPROVED** [review]
yay lets goooooooo

**Pull request merged**
